Makefile

```diff
@@ -0,0 +1,14 @@
+build:
+	pipenv install --dev
+
+lint:
+	pipenv check ./app ./tests
+
+test: lint
+	pipenv run pytest ./tests
+
+extract:
```

Comment:
I like the Makefile, not sure about these last 2 commands though.  Given you have to edit them to be useful, it might just be easier to run the original commands, you could also potentially choose which schema to use with an env var?

Reply:
I'm not sure if there's a way (more than likely is). 
Could set something like $SCHEMA and run `make extract schema`

Reply:
@dcdarrell9 This stuff should have been put into a different PR for context and clarity

Reply:
@shahi645 Maybe, but it's 1 extra file on a repo that doesn't get used too often that helps with running the application easier

Reply:
i don't disagree with that, but its a change we would probably want documenting, so can be referenced later on, i.e. better in its own PR.

Reply:
same could be said for the travis integration. Maybe put the two of them into a PR?